corehq/apps/users/models.py

```diff
@@ -213,6 +213,7 @@ class HqPermissions(DocumentSchema):
     manage_data_registry_list = StringListProperty(default=[])
     view_data_registry_contents = BooleanProperty(default=False)
     view_data_registry_contents_list = StringListProperty(default=[])
+    manage_attendance_tracking = BooleanProperty(default=False)
```

Comment:
It seems like everything is working fine when I added this line. Existing `HqPermissions` gets a default value for this field, as per the `default=False` spec. Since all the tests pass and I don't see any errors by navigating and playing around with the roles, I presume only adding this line here in order to add a new permission is fine?
![image](https://user-images.githubusercontent.com/64970009/218465102-c95ba2f0-1d5b-40d2-9b5e-45d783fce804.png)

Reply:
This is gold, thanks @esoergel

Reply:
whoops, meant to link this PR:
https://github.com/dimagi/commcare-hq/pull/31659